src/Util/FileHelper.php

```diff
@@ -25,6 +25,10 @@ public static function normalizeSystemPath(string $originalPath): string
             [, $scheme, $path] = $matches;
         }
 
-        return ($scheme !== null ? $scheme . '://' : '') . str_replace(['/', '\\'], [DIRECTORY_SEPARATOR, DIRECTORY_SEPARATOR], $path);
+        if (DIRECTORY_SEPARATOR === '\\') {
```

Comment:
How is this mutation-tested, since we don't run MT on windows? 🤔

Reply:
Also, I wonder if we should just normalize to windows path names regardless?

Reply:
> How is this mutation-tested, since we don't run MT on windows?

You're right, it's not. I can probably add ignore annotation...

> Also, I wonder if we should just normalize to windows path names regardless?

It will report errors somewhere, where we check differences between BR and native reflection.

Reply:
> It will report errors somewhere, where we check differences between BR and native reflection.

Good point!